freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -123,10 +132,17 @@ public CloudStack convert(Stack stack, Collection<String> deleteRequestedInstanc
     public CloudInstance buildInstance(Stack stack, InstanceMetaData instanceMetaData, InstanceGroup instanceGroup,
             StackAuthentication stackAuthentication, Long privateId, InstanceStatus status) {
         ImageEntity imageEntity = imageService.getByStack(stack);
-        return buildInstance(instanceMetaData, instanceGroup, stackAuthentication, privateId, status, imageEntity.getImageName(), stack.getEnvironmentCrn());
+        return buildInstance(stack,
+                instanceMetaData,
+                instanceGroup,
+                stackAuthentication,
+                privateId,
+                status,
+                imageEntity.getImageName(),
+                stack.getEnvironmentCrn());
```

Comment:
Nit: Since we are passing `stack` anyway, `stack.getEnvironmentCrn()` could be also removed as an explicit argument.